bugwarrior/services/github.py

```diff
@@ -324,16 +324,23 @@ def get_owner(self, issue):
         if issue[1]['assignee']:
             return issue[1]['assignee']['login']
 
+    def filter_issues(self, issue):
+        repo, _ = issue
+        return self.filter_repo_name(repo.split('/')[-3])
```

Comment:
There is a part of me that hopes github will support nested repos under orgs someday :) .